react/week4/github-reasecher/src/context/UserSearchResults.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { searchContext } from './createContext'
+
+const UserName = (props) => {
+    return (
+        <div>
+            {props.username}
+        </div>)
+}
+
+const UserSearchResults = () => {
+    const gitUserNames = React.useContext(searchContext)
+    return (
+        <div>
+            {gitUserNames.users.map(user => <UserName key={user.id ? user.id : 1} username={user.login ? user.login : user} />)}
```

Comment:
Just a suggestion for making it a little bit shorter.
```suggestion
            {gitUserNames.users.map(user => <UserName key={user.id ?? 1} username={user.login ?? user} />)}
```